pyproject.toml

```diff
@@ -150,7 +150,7 @@ coveralls = "^2.0.0"
 towncrier = "^19.2.0"
 toml = "^0.10.0"
 pep440-version-utils = "^0.3.0"
-pydoc-markdown = "^3.5.0"
+pydoc-markdown = "^3.5.1"
```

Comment:
this requires https://github.com/NiklasRosenstein/pydoc-markdown/pull/156 to be merged and released to pypi

Reply:
that one seems to be released, should we update?